sympy/core/numbers.py

```diff
@@ -947,6 +947,7 @@ def __new__(cls, num, dps=None, prec=None, precision=None):
                             issue=12820,
                             deprecated_since_version="1.1").warn()
             dps = prec
+        del prec
```

Comment:
Can you add a comment here that the del is there to avoid using the deprecated variable.

Reply:
Otherwise +1 to this.